qucs-doc/report/EKV26/Makefile.am

```diff
@@ -29,6 +29,8 @@ EXTRA_DIST = $(SOURCES)
 SOURCES = EKV26.tex content.tex
 TARGETS = EKV26.pdf
 
+export TEXINPUTS=$(srcdir):$(srcdir)/..:
```

Comment:
commit 641ce37e88f5c515288739e2983e71dfdb112590 already did it right, why do you deviate from it?
global variables are a bad idea, and ideas are prone to cargo cult.

and (sorry for being pedantic, but) please include my commit in your PR if you write "...expanded #524".